src/main/java/io/getstream/chat/java/models/App.java

```diff
@@ -894,11 +888,20 @@ public static AppRevokeTokensRequest revokeTokens(@Nullable Date revokeTokensIss
    * @param signature the signature
    * @return true if the signature is valid
    */
-  public boolean verifyWebhook(String body, String signature) {
-    String apiSecret =
-        System.getenv("STREAM_SECRET") != null
-            ? System.getenv("STREAM_SECRET")
-            : System.getProperty("STREAM_SECRET");
+  public boolean verifyWebhook(@NotNull String body, @NotNull String signature) {
+    return verifySignature(body, signature);
+  }
+
+  /**
+   * Validates if hmac signature is correct for message body. Can be used for webhook verification
+   *
+   * @param apiSecret the secret key
+   * @param body the message body
```

Comment:
could you add clarification about `it shouldn't be changed from the wire format`?